src/gui/optionsdialog.cpp

```diff
@@ -461,10 +461,7 @@ void OptionsDialog::saveBehaviorTabOptions() const
     pref->setLocale(locale);
 
 #ifdef Q_OS_WIN
-    if (const QVariant systemStyle = m_ui->comboStyle->currentData(); systemStyle.isValid())
-        pref->setStyle(systemStyle.toString());
-    else
-        pref->setStyle(m_ui->comboStyle->currentText());
+    pref->setStyle(m_ui->comboStyle->currentData().toString());
```

Comment:
Wouldn't be better that the setting value in config file has the same case as seen on the combobox? Note that `style` in `QApplication::setStyle(const QString &style)` is case insensitive.

```suggestion
    pref->setStyle(m_ui->comboStyle->currentText());
```